bin/setup

```diff
@@ -14,20 +14,26 @@ FileUtils.chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts "== Installing dependencies =="
+  system("yarn install")
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   FileUtils.cp "config/database.yml.sample", "config/database.yml"
-  # end
+  puts "\n== Copying development settings file =="
+  system! "sed 's/secret: secret/secret: persona/' config/settings/development.yml > config/settings/development.local.yml"
```

Comment:
Should we consider users on non unix-like operating systems? 🤔

Reply:
In theory, yes!

I'll update to print a warning if on Windows. I can't test a sed-like command and it isn't worth doing either.